Source/Example/Catalog/Web/CartController.cs

```diff
@@ -50,7 +50,10 @@ public void Add([FromBody]AddItemToCart command)
             var cartId = _carts.GetCartIdForCurrentUser();
             var price = _pricing.GetForProduct(command.Product);
 
-            _eventEmitter.Emit(Feature, new ItemAddedToCart
+            var eventProcessors = _serviceProvider.GetService(typeof(IInstancesOf<IKnowAboutEventProcessors>)) as IInstancesOf<IKnowAboutEventProcessors>;
```

Comment:
I see the eventProcessors and es variables are unused, should we add a comment that they exists to demonstrate how you could retrieve it from a the serviceprovider? Just so that no one removes them.

Reply:
Forgot to check the example app...  :) Very sloppy of me..  On the case.